lib/ex_unit/lib/ex_unit/runner.ex

```diff
@@ -132,8 +133,9 @@ defmodule ExUnit.Runner do
     tests = shuffle(config, tests)
     include = config.include
     exclude = config.exclude
+    test_ids = config.only_test_ids
```

Comment:
We need to document this and the failure_manifest_file in lib/ex_unit.ex.

Reply:
Done.